content/tutorials/manage-the-rippled-server/installation/build-run-rippled-macos.md

```diff
@@ -36,11 +36,15 @@ For development purposes Ripple recommends running `rippled` as your own user, n
             ./bootstrap.sh
             ./b2 cxxflags="-std=c++14"
 
-0. Ensure that your `BOOST_ROOT` environment variable points to the directory created by the Boost installation. To find your Boost install directory, use `brew info boost`. Put this environment variable in your `.bash_profile` file so it's automatically set when you log in. For example:
+0. Ensure that your `BOOST_ROOT` environment points to the directory created by the Boost installation:
+        
+      1. To find your Boost directory, use `pwd` if you installed the Boost manually or use `brew info boost` if you installed the Boost with Homebrew.
```

Comment:
maybe we should take the opportunity to update this to `brew --prefix boost` since that directly returns the boost root (you don't have to read through all the info output)